### PR TITLE
Added pretty links for citizens and teams

### PR DESCRIPTION
**Pull request by namedotget**

### CODE REVIEW

**Review by pmoncada — CHANGES_REQUESTED** [review]
Nice - Looks great!

The only thingg that I'd add would be good to add some unit tests. In particular around checking that it generates a unique ID if there's a clash between pretty links / other potential corner cases with the pretty links

File: ui/pages/network.tsx [hunk 1]
```diff
@@ -330,10 +337,25 @@ export async function getStaticProps() {
...
+  const prettyLinks = {
+    team: {},
+    citizen: {},
+  }
+  const teamPrettyLinkData = filteredValidTeams.map((nft: any) => ({
+    name: nft?.metadata?.name,
+    id: nft?.metadata?.id,
+  }))
+  const { idToPrettyLink: teamIdToPrettyLink } =
+    generatePrettyLinks(teamPrettyLinkData)
```
Comment (pmoncada) [inline ui/pages/network.tsx:350]: The way I'm understanding this we're regenerating all the links every time we load the page...

Would it make sense to just store the unique text ID for each in tableland? That way we're not doing an O(N) computation each time we render the page, and just doing it once when a team is generated?

Especially as we scale with more Citizens and Teams, it might make sense to just store this data v.s. generating  it on the fly every time. Also, this could cause some issues where if a team is deleted, then the "link" for a team further down the list might get changed (there's no longer a conflict), so it's better if these links are just static after being generated, versus dynamic and calculated each time.

**Review by pmoncada — CHANGES_REQUESTED** [review]

File: ui/pages/network.tsx (hunk 1 above)
Comment (namedotget) [reply ui/pages/network.tsx:350]: If the pretty link already exists then the tokenId is appended to the end of the pretty link

**Review by pmoncada — APPROVED** [review]
Looks good to me - it's too bad all the links will now have a number, but I think this method makes things more straightforward on our end.
